packages/backend/src/util/environment.ts

```diff
@@ -1,3 +1,13 @@
-import * as env from 'env-var'
+import { config } from 'dotenv'
+import { from } from 'env-var'
+const env = from(process.env, {})
+config()
 
-export const BACKEND_PORT = env.get('BACKEND_PORT').default(3300).asInt()
```

Comment:
Same as in #34

Reply:
This branch based on branch used ín #34 I dont know why or how happend

Reply:
Cause you oppend up the PR against the main, and that PR also not merged yet. So the changes against the main still apper. If it's based on that branch, I don't think a conflict will appear, but be careful when merging

Reply:
Ok thanks for advice